public/assets/img/RIFOS RNS MANAGER.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<!-- saved from url=(0073)file:///X:/PROYECTOS/RIF/RNS_Manager/V6/TEMPLATE/admin-resolution-qr.html -->
```

Comment:
You sure that you want to commit with this comment?

Also why is this html file in `public/assets/img`?